src/components/Api.js

```diff
@@ -0,0 +1,88 @@
+import Card from "./Card.js";
+import UserInfo from "./UserInfo.js";
+
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  _handleResponse(res) {
```

Comment:
Great that you've implemented a  method for processing server response, so the code that checks the server response and converts it from JSON  is reused

Reply:
thanks!